Makefile

```diff
@@ -53,3 +54,6 @@ install:
 uninstall:
 	/bin/rm -f $(DESTDIR)/$(MODULE_DESTDIR)/nv_peer_mem.ko
 	if [ ! -n "$(DESTDIR)" ]; then $(DEPMOD) -r -ae $(KVER);fi;
+
+gen_nv_symvers:
+	$(PWD)/create_nv.symvers.sh $(KVER)
```

Comment:
Right. Again I missed that. I accidentally looked at the master version.  Thanks!